src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -611,15 +611,27 @@ private function injectAlignmentPlaceholders(Tokens $tokens, $startAt, $endAt, $
             }
 
             if ($token->equals('(')) {
-                $index = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index);
+                $end = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index);
 
-                continue;
+                if (false !== strpos($tokens->generatePartialCode($index, $end), 'function (')) {
```

Comment:
This will only work with anonymous functions declarations that stricly have one space between `function` and the opening parenthesis. There might be newlines, comments, or nothing at all. IMO, having a condition on what the generated code looks like is not a good approach.